### PR TITLE
Trigger downstream environment repository build

### PR DESCRIPTION
**Pull request by giorgiosironi**
The `environments` repository deploys the latest version of each project to the `unstable` environment. It will be theoretically possible to specify a particular Docker image tag to deploy, but left for future development.

The build being triggered looks like this:

(already added the TRAVIS_TOKEN environment variable)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,3 +27,9 @@ after_success:
 
 after_script:
     - docker-compose -f docker-compose.yaml down
+
+deploy:
+    provider: script
+    script: .travis/downstream.sh $TRAVIS_COMMIT
```
Comment (thewilkybarkid) [inline .travis.yml]: `downstream` is a bit too generic, might end up with multiple downstreams.

File: .travis/downstream.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+body='{
+	"request": {
```
Comment (thewilkybarkid) [inline .travis/downstream.sh]: Spaces

File: .travis/downstream-environments.sh [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+body='{
+    "request": {
+        "branch":"master",
+        "message":"browser new version: '"$1"'"
```
Comment (thewilkybarkid) [inline .travis/downstream-environments.sh]: Is this both human-readable and parsable? Could be `Update Browser to 12345`?

Comment (giorgiosironi) [reply .travis/downstream-environments.sh]: Human-oriented, will update

**Pull request merged**